donut/modules/newsgroups/helpers.py

```diff
@@ -65,14 +65,22 @@ def get_user_actions(user_id, group_id):
         cursor.execute(query, (user_id, group_id))
         res = cursor.fetchall()
     actions = {'send': False, 'control': False, 'receive': False}
-    if not res:
+    anyone_can_send_query = """SELECT anyone_can_send FROM groups
+        WHERE group_id=%s"""
+    with flask.g.pymysql_db.cursor() as cursor:
+        cursor.execute(anyone_can_send_query, (group_id))
+        anyone_can_send = cursor.fetchone()
+
+    if not res and not anyone_can_send:
```

Comment:
Probably don't need this special case, but doesn't really matter

Reply:
It bugged me when the dropdown had a certain newsgroup and then when you tried to send something, it gives a permission error @.@

Reply:
No, I just meant that the `if not res and not anyone_can_send: return actions` is not necessary